src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+.headerBreadcrumbs {
+  margin-left: 0;
+  color: $ouiColorPrimary;
```

Comment:
q: should we use oui or eui vars in OSD?

Reply:
`eui` for sure.